tox.ini

```diff
@@ -114,6 +114,7 @@ deps =
     django-2.2: Django>=2.2,<2.3
     django-3.0: Django>=3.0,<3.1
     django-3.1: Django>=3.1,<3.2
+    django-3.2: Django>=3.1,<3.3
```

Comment:
Why did we introduce this dependency?

Reply:
because django 4 came out and this 3.2 would resolve to 4 instead

Reply:
Shouldn't it be `>=3.2` instead?
```suggestion
    django-3.2: Django>=3.2,<3.3
```
I'm not familiar with this, but how would django v4 be resolved without this change? Just asking.

Reply:
indeed, typo, thx for spotting